apps/condo/domains/ticket/schema/UserTicketCommentReadTime.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Generated by `createschema ticket.UserTicketCommentReadTime 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE; readResidentCommentAt:DateTimeUtc;'`
+ */
+
+const { Relationship, DateTimeUtc } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/UserTicketCommentReadTime')
+
+
+const UserTicketCommentReadTime = new GQLListSchema('UserTicketCommentReadTime', {
+    schemaDoc: 'Time when a comment from a resident was last read by a specific user in a specific ticket',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        user: {
+            schemaDoc: 'The user who read the comment',
+            type: Relationship,
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        ticket: {
+            schemaDoc: 'Ticket in which the user read the comment',
+            type: Relationship,
+            ref: 'Ticket',
+            isRequired: true,
+            knexOptions: { isNotNullable: true },
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        readResidentCommentAt: {
```

Comment:
You also need the second one filled for non resident comments

Reply:
added